src/routes/account/account.controller.ts

```diff
@@ -0,0 +1,64 @@
+import { Request, Response } from 'express';
+import { Account } from '../../schemas/account';
+
+export async function getAccount(
+  req: Request,
+  res: Response
+) {
+  try{
+    const docs =  await Account.findById(req.query.userId)
+    if(!docs) {
+      res.status(404);
+      return res.json();
```

Comment:
```suggestion
      return res.status(404).json();
```

Du kannst das hier direkt aneinander hängen, dann liest es sich leichter, ist aber kein muss